### PR TITLE
Add target information to additional-info comment

### PR DESCRIPTION
**Pull request by patrick-rivos**

### CODE REVIEW

File: .github/workflows/run_checks.yaml [hunk 1]
```diff
@@ -112,6 +112,16 @@ jobs:
           printf "Logs can be found in the associated Github Actions run: https://github.com/ewlu/gcc-precommit-ci/actions/runs/${{ github.run_id }}\n" >> issue.md
           printf "## Patch information\n" >> issue.md
           head -n 3 patches/${{ inputs.patch_name }} >> issue.md
+          printf "## Build Targets" >> issue.md
```
Comment (ewlu) [inline .github/workflows/run_checks.yaml:115]: `printf` won't add a newline at the end like `echo` does. Needs a `\n` at the end

File: .github/workflows/run_checks.yaml [hunk 2]
```diff
@@ -112,6 +112,16 @@ jobs:
...
+          echo "Some targets are built as multilibs. If a build target ends with `multilib`, please refer to the table below to see all the targets within that multilib." >> issue.md
```
Comment (ewlu) [inline .github/workflows/run_checks.yaml:116]: the backticks surrounding multilib need to be escaped

File: .github/workflows/run_checks.yaml [hunk 3]
```diff
@@ -112,6 +112,16 @@ jobs:
...
+          printf "|Target name|`-march` string|" >> issue.md
```
Comment (ewlu) [inline .github/workflows/run_checks.yaml:117]: need `\n` and escaped backticks for

**Review by ewlu — CHANGES_REQUESTED** [review]
Need to update all printf locations

**Comment by patrick-rivos** [comment]
https://github.com/ewlu/gcc-precommit-ci/actions/runs/6696610663

**Review by ewlu — APPROVED** [review]
LGTM

**Pull request merged**